frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "craco-less": "1.20.0",
     "framer-motion": "6.3.3",
     "moment": "^2.29.3",
+    "package.json": "^2.0.1",
```

Comment:
I didn't change anything here so it may have come by merging in main or another branch.

Reply:
I'll delete and re-merge dev into this branch.

Reply:
@imagine-hussain this could have happened if you ran `npm install` wierdly, which is what i suspect - you can probably remove this from your package.json and run npm install again to fix your package-lock.json.